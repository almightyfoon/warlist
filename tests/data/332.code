c8501b_-0j404B5l4zfrfr5454eE4_4_525w525w___-0p5C4Qcn4H4Hg4g45g5g5e5fi7