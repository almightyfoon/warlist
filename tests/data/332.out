https://conflictchamber.com/#c8501b_-0j404BmJ4zfrfr5454eE4_4_mj5wmj5w___-0p5C4Qcn4H4Hg4g45g5g5e5fi7

Circle Army - 72 / 75 points
[Theme] The Devourer's Host

[Krueger 1] Krueger the Stormwrath [+28]
 - Gorax Rager [7]
 - Warpwolf Stalker [17]
 - Wild Argus [7]
Bloodweaver Night Witch [0(4)]
Bloodweaver Night Witch [0(4)]
Tharn Ravager Shaman [0(4)]
Tharn Ravager Shaman [4]
Death Wolves [9]
Tharn Bloodweavers [8]
Tharn Bloodweavers [8]
Tharn Ravagers (max) [15]
 - Tharn Ravager Chieftain [5]
Tharn Ravagers (max) [15]
 - Tharn Ravager Chieftain [5]

Circle Army - 75 / 75 points
[Theme] The Bones of Orboros

[Baldur 2] Baldur the Stonesoul [+30]
 - Megalith [20]
 - Woldwrath [37]
 - Woldwyrd [9]
 - Woldwyrd [9]
Blackclad Stoneshaper [0(3)]
Blackclad Stoneshaper [3]
Blackclad Wayfarer [0(4)]
Blackclad Wayfarer [0(4)]
Sentry Stone & Mannikins [5]
Shifting Stones [3]
Celestial Fulcrum [19]
